src/enacrestic/__init__.py

```diff
@@ -45,6 +47,7 @@
 ENACRESTIC_PREF_FOLDER = os.path.expanduser('~/.enacrestic')
 RESTIC_USER_PREFS = {
     'FILESFROM': os.path.join(ENACRESTIC_PREF_FOLDER, 'bkp_include'),
+    'EXCLUDEFILES': os.path.join(ENACRESTIC_PREF_FOLDER, 'bkp_exclude'),
```

Comment:
As restic's parameter is `--exclude-file` (singular), could you rename this variable to `EXCLUDEFILE` please ?

Reply:
Done